cli/src/installers/dependencyVersionMap.ts

```diff
@@ -4,7 +4,7 @@
  */
 export const dependencyVersionMap = {
   // NextAuth.js
-  "next-auth": "^4.24.7",
+  "next-auth": "beta",
```

Comment:
should probably pin this while it's still in beta.

```suggestion
  "next-auth": "5.0.0-beta.25",
```